.zuul.d/collect-logs.yml

```diff
@@ -37,6 +37,17 @@
                   cp -ra {{ ansible_user_dir }}/ci-framework/logs . ;
                   cp -ra {{ ansible_user_dir }}/ci-framework/artifacts . ;
 
+            - name: Check for Zuul inventory file
+              ansible.builtin.stat:
+                path: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}/scenarios/centos-9/zuul_inventory.yml"
+              register: cifmw_zuul_state
+
+            - name: Copy Zuul inventory file
+              when: cifmw_zuul_state.stat.exists | bool
+              ansible.builtin.copy:
+                src: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}/scenarios/centos-9/zuul_inventory.yml"
+                dest: "{{ ansible_user_dir }}/ci-framework/logs"
```

Comment:
oh, right, you may want to add:
`mode: 0644` 

else it's created with 0600 - and may prevent proper read.